### PR TITLE
Feature sprint1 api adjustments 2

### PR DESCRIPTION
**Pull request by petemade**
More entity adjustments surrounding sprint 1 home care work

#### _Checklist_

- [ ] Time slot shift adjustments
- [ ] Home care service adjustments to account for night services
- [ ] Removing unrequired code

### CODE REVIEW

**Review by FurkanKayar55 — APPROVED** [review]

File: LBH.AdultSocialCare.Api/Startup.cs [hunk 1]
```diff
@@ -356,7 +354,6 @@ public static void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             IMapper mapper = app.ApplicationServices.GetService<IMapper>();
             ApiToDomainFactory.Configure(mapper);
             DomainToEntityFactory.Configure(mapper);
-            EntityToDomainFactory.Configure(mapper);
```
Comment (dunnudge) [inline LBH.AdultSocialCare.Api/Startup.cs:359]: There is automapper in this file. Will this not cause a problem mapping entities to domains?

File: LBH.AdultSocialCare.Api/V1/Infrastructure/DatabaseContext.cs [hunk 2]
```diff
@@ -34,7 +33,7 @@ public DatabaseContext(DbContextOptions options)
         public DbSet<HomeCarePackageSlots> HomeCarePackageSlots { get; set; }
         public DbSet<Users> Users { get; set; }
         public DbSet<Clients> Clients { get; set; }
-        public DbSet<Status> Status { get; set; }
+        public DbSet<PackageStatus> Status { get; set; }
```
Comment (dunnudge) [inline LBH.AdultSocialCare.Api/V1/Infrastructure/DatabaseContext.cs:36]: The dbset/table can be named PackageStatus as well. Would make more sense for someone working on the DB

**Review by dunnudge — APPROVED** [review]
Looking good adding some comments for consideration. Noticed emphasis on more descriptive property names. Nice.

File: LBH.AdultSocialCare.Api/Startup.cs (hunk 1 above)
Comment (petemade) [reply LBH.AdultSocialCare.Api/Startup.cs:359]: Sorry yes - I had deleted the file, will revert this

File: LBH.AdultSocialCare.Api/V1/Infrastructure/DatabaseContext.cs (hunk 2 above)
Comment (petemade) [reply LBH.AdultSocialCare.Api/V1/Infrastructure/DatabaseContext.cs:36]: Thanks Duncan

**Pull request merged**